src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -252,7 +252,8 @@ The .NET Foundation licenses this file to you under the MIT license.
       <IlcArg Condition="$(TargetArchitecture) != ''" Include="--targetarch:$(TargetArchitecture)" />
       <IlcArg Condition="'$(TargetArchitecture)' == 'wasm'" Include="--targetos:wasm" />
       <IlcArg Condition="$(IlcMultiModule) == 'true'" Include="--multifile" />
-      <IlcArg Condition="$(Optimize) == 'true'" Include="-O" />
+      <!-- TODO-LLVM: when optimize is on, the clrjit is asserting so turn off to get the CI working -->
+      <IlcArg Condition="$(Optimize) == 'true' and '$(TargetArchitecture)' != 'wasm'" Include="-O" />
```

Comment:
Hmm, maybe this would be a little more visible to turn `Optimize` off at the top of the file (where `<NativeCodeGen>llvm</NativeCodeGen>` is defined), it seems very easy to miss and spend time wondering "why are optimizations off".

Reply:
Thanks, yes that's better